app/models/container_template.rb

```diff
@@ -12,4 +12,51 @@ class ContainerTemplate < ApplicationRecord
   serialize :objects, Array
 
   acts_as_miq_taggable
+
+  def instantiate(params, project = nil)
+    project ||= container_project.name
+    processed_template = process_template(ext_management_system.connect,
+                                          :metadata   => {
+                                            :name      => name,
+                                            :namespace => project
+                                          },
+                                          :objects    => objects,
+                                          :parameters => params
+                                         )
+    create_objects(processed_template['objects'], project) if processed_template
```

Comment:
@zakiva are the created objects after refresh stored in `ContainerTemplate#objects` field? Will processed_template be saved too? Can you explain the relationships among original template, processed_template, and created objects?